jobs/JGLOBAL_ATMOS_PREP_SFC

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/jjob_header.sh" -e "prep_sfc" -c "base prep_sfc"
+
+##############################################
+# Set date and cycle
+##############################################
+
+# shellcheck disable=SC2153
+GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
+export gcycle="t${gcyc}z"
```

Comment:
I understand this is a convenient short-hand, but `${gcyle}` is no less characters than `t${gyc}z`.  IMO, it just is a re-definition and we should avoid it.

Reply:
Updated @ f40491231. I also updated some `${cycle}="t${cyc}z` references.